pysteps/blending/steps.py

```diff
@@ -90,6 +90,7 @@ def forecast(
     conditional=False,
     probmatching_method="cdf",
     mask_method="incremental",
+    create_smooth_radar_mask=False,
```

Comment:
can we add a test case setting this flag as True?

Reply:
Some tests still need to be written.